compiler/repl-service/protos/BUILD.bazel

```diff
@@ -5,22 +5,22 @@ load("//bazel_tools:haskell.bzl", "da_haskell_library", "da_haskell_test")
 load("//bazel_tools:proto.bzl", "proto_gen")
 
 proto_library(
-    name = "repl_service_proto",
+    name = "repl_service_proto_lib",
```

Comment:
renamed for cache invalidation purposes. Something is very wrong here but doesn’t seem sensible to block the backport work on debugging caching.